terraform/modules/data-share-service/iam.tf

```diff
@@ -1,6 +1,33 @@
 resource "aws_iam_role" "ecs_execution" {
   name               = "${var.environment}-execution-role"
   assume_role_policy = data.aws_iam_policy_document.assume_role_policy.json
+
+  inline_policy {
```

Comment:
Could we make this its own policy rather than an inline please :)

Reply:
Yes I just can't work out how to do that!